rero/rero-ils
# Circulation: integration of invenio-circulation APIs

* NEW makes invenio circ APIs available

Signed-off-by: Aly Badr <aly.badr@rero.ch>

To test:
- ./scripts/bootstrap
- ./scripts/setup
- ./run_tests.sh

To test some circulation transactions, check examples in Taiga:
https://tree.taiga.io/project/rero21-reroils/task/708?kanban-status=1224894
